openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -63,12 +111,13 @@ def _post_body(self, user, client, token_type=None):
 
 
 @ddt.ddt
-class TestAccessTokenView(mixins.AccessTokenMixin, _DispatchingViewTestCase):
+class TestAccessTokenView(mixins.AccessTokenMixin, _DispatchingViewTestCase, AccessTokenLoginMixin):
     """
     Test class for AccessTokenView
     """
     def setUp(self):
         self.url = reverse('access_token')
+        self.login_with_access_token_url = reverse("login_with_access_token")
```

Comment:
@chrisndodge You should declare the Mixin class before the concrete class in the list of base classes.  And the `setUp` implementation on the Mixin class should make sure to call `super` (even if it doesn't have any base classes).  This will ensure that all class ancestors are called through the `super`-chain.